webdriver-spec.html

```diff
@@ -4064,13 +4064,12 @@
 
  <li><p>Let <var>result</var> be an empty JSON <a>List</a>.
 
- <li><p>For each <var>element</var> in <var>elements returned</var>:
-
-  <ul>
-   <li><p>Append to <var>result</var>
-    the <a data-lt="JSON Serialization of an element">JSON Serialization</a>
-    of the <var>element</var>.
-  </ul>
+ <li><p>For each <var>element</var> in <var>elements returned</var>,
```

Comment:
We could, or do the check in the `Find` algorithm, remove the text from `link text` and make it so that if we add something new (there's an open issue for finding by plain text) we don't need to remember to add this check.